app/interface/api.ts

```diff
@@ -18,6 +18,37 @@ export class Api {
         },
     ) {}
 
+    sendSummaryFeedbackHandler: Handler = async (params) => {
+        try {
+            const request = new Request(params);
+            var { to, context }: MailSummaryFeedback = {
+                to: request.parseString('to'),
+                context: {
+                    nombre: request.parseString('student.full_name'),
+                    ejercicios: request.parseString('summary.ejercicios'), // TODO: parsear a array
```

Comment:
Creo que se puede usar este workaround
```suggestion
                    ejercicios: request.map('summary.ejercicios', (req) => req.parseString('toString()')),
```